accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-anchor-rules.ts

```diff
@@ -63,6 +63,8 @@ let a11yRulesAnchor: Rule[] = [{
     context: "dom:a[target],dom:area[target],dom:base[target]",
     run: (context: RuleContext, options?: {}): RuleResult | RuleResult[] => {
         const ruleContext = context["dom"].node as Element;
+        //skip the rule
+        if (RPTUtil.isNodeHidden(ruleContext)) return null;
```

Comment:
I think we need to check for a tabindex=-1 also. Interactive content shouldn't be hidden, so it would be invalid to be aria-hidden without a negative tabindex.

Reply:
@tombrunet  the focus check is covered by another rule "aria_hidden_focus_misuse": element with aria-hidden shouldn't contain any focusable content.  Do we need to check again here?

Reply:
@shunguoy We don't silence checks unless they're redundant. In this case, it's checking two different things, so the fact that it should also trigger isn't relevant. If they choose to ignore the focus while hidden error, they won't know that they also have this recommendation.

Reply:
removed